browse/services/util/response_headers.py

```diff
@@ -12,12 +12,17 @@
 """Days of the week publish happens: Sunday-Thursday."""
 
 
-def guess_next_update_utc(dt: datetime = datetime.now(timezone.utc)) \
-        -> Tuple[datetime, bool]:
+def guess_next_update_utc(dt: Optional[datetime] = None) -> Tuple[datetime, bool]:
```

Comment:
Changed default parameter to none so it gets set to now.

When set in the function definition, dt get set to the time the function definition is evaluated.